README.md

```diff
@@ -80,15 +81,21 @@ You can setup storages either at zenoh router startup via a configuration file,
               volume: {
                 id: "influxdb",
                 // the database name within InfluxDB
+                // NOTE: For influxdb v2.x, the influxdb rust library currently doesn't provide for creating the database so this db should already exist
                 db: "zenoh_example",
+                
                 // if the database doesn't exist, create it
-                create_db: true,
+                // NOTE: For influxdb v2.x, the influxdb rust library currently doesn't provide for creating the database so this value sshould be kept false
+                create_db: false,
                 // strategy on storage closure
-                on_closure: "drop_db",
+                // NOTE: For influxdb v2.x, the influxdb rust library currently doesn't provide for dropping the database so this should NOT be kept to drop_db
```

Comment:
Same comment as above.
Weird wording... doesn't provide for? Did you mean doesn't provide SUPPORT for?